lexicons/tools/ozone/server/getConfig.json

```diff
@@ -32,14 +33,19 @@
     "serviceConfig": {
       "type": "object",
       "properties": {
-        "configured": {
-          "type": "boolean"
-        },
         "url": {
           "type": "string",
           "format": "uri"
         }
       }
+    },
+    "viewerConfig": {
+      "type": "object",
+      "properties": {
+        "role": {
+          "type": "string"
+        }
```

Comment:
We'll want to remember to sync this with your work in #2460 👍 

Should we look ahead and at least start by using compatible role values with the ones used in that PR?